addons/gunbag/functions/fnc_BIArsenalClose.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: mjc4wilton
+ * Handle BI arsenal closing
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_gunbag_fnc_BIArsenalClose
+ *
+ * Public: No
+ */
+
+if (!isNil QGVAR(arsenalCache)) then {
+    (backpackContainer player) setVariable [QGVAR(gunbagWeapon),GVAR(arsenalCache), true];
```

Comment:
```sqf
    backpackContainer player setVariable [QGVAR(gunbagWeapon), GVAR(arsenalCache), true];
```
Can you open the arsenal while remote controlling AI as Zeus?

In that case, this variable would be set on the wrong unit, as the `player` command is used, instead of a entity being passed around by the functions.